src/ripple/app/tx/impl/ApplyContext.cpp

```diff
@@ -69,4 +71,57 @@ ApplyContext::visit (std::function <void (
     view_->visit(base_, func);
 }
 
+template<std::size_t... Is>
+TER
+ApplyContext::checkInvariantsHelper(TER terResult, std::index_sequence<Is...>)
+{
+    if (view_->rules().enabled(featureEnforceInvariants))
+    {
+        auto const& section = app.config().section("check_invariants");
+        if (get<bool> (section, "enabled", true))
+        {
+            auto checkers = getInvariantChecks();
```

Comment:
This value isn't documented in the example config file. Should it be?

Also, if I'm reading this right, the syntax is:
```
[check_invariants]
enabled=true
``` 

I can't think of any reason why `[check_invariants]` would get multiple entries, so making the syntax complex for complexities sake doesn't sit well with me. At the very least, I'd suggest:
```
[check_invariants]
true
``` 

But since we expect this option to only be tweaked for debugging purposes, why can't we add it as a command-line flag like `--no-invariant-checking`?